client/components/Map/Map.jsx

```diff
@@ -45,6 +45,10 @@ import MapSearch from './controls/MapSearch';
 // import MapMeta from './controls/MapMeta';
 
 import RequestDetail from './RequestDetail';
+import {
```

Comment:
We can remove this block since unused
`import {
  call,
} from 'redux-saga/effects';
import axios from 'axios';`